src/include/cc.h

```diff
@@ -36,11 +36,12 @@ constexpr bool seqno_gt(uint32_t a, uint32_t b) {
  */
 // TODO(ilias): First-cut implementation. Needs a lot of work.
 struct Pcb {
-  static constexpr std::size_t kInitialCwnd = 32;
+  static constexpr std::size_t kInitialCwnd = 128; // start from 128 as inital window
```

Comment:
Delete comment, since `kInitialCwnd` already clarifies that this is the initial window.